docs/modules/getting_started/pages/first_steps.adoc

```diff
@@ -0,0 +1,143 @@
+
+`connections.sqlalchemyDatabaseUri` must contain the connection string to the SQL database storing the Airflow metadata.
+
+`connections.celeryResultBackend` must contain the connection string to the SQL database storing the job metadata (in the example above we are using the same postgresql database for both).
+
+`connections.celeryBrokerUrl` must contain the connection string to the Redis instance used for queuing the jobs submitted to the airflow worker(s).
+
+The `adminUser` fields are used to create an admin user.
+Please note that the admin user will be disabled if you use a non-default authentication mechanism like LDAP.
+
+=== Airflow
+
+An Airflow cluster is made of up three components:
+
+- `webserver`: this provides the main UI for user-interaction
+- `workers`: the nodes over which the job workload will be distributed by the scheduler
+- `scheduler`: responsible for triggering jobs and persisting their metadata to the backend database
+
+Create a file named `airflow.yaml` with the following contents:
+
+[source,yaml]
+----
+include::example$code/airflow.yaml[]
+----
+
+And apply it:
+
+----
+include::example$code/getting_started.sh[tag=install-airflow]
+----
+
+Where:
+
+- `metadata.name` contains the name of the Airflow cluster
+- the label of the Docker image provided by Stackable must be set in `spec.version`
+- `spec.statsdExporterVersion` must contain the tag of a statsd-exporter Docker image in the Stackable repository.
+- `spec.executor`: this setting determines how the cluster will run (for more information see https://airflow.apache.org/docs/apache-airflow/stable/executor/index.html#executor-types): the `CeleryExecutor`
+is the recommended setting although `SequentialExecutor` (all jobs run in one process in series) and `LocalExecutor`
+(whereby all jobs are run on one node, using whatever parallelism is possible) are also supported
+- the `spec.loadExamples` key is optional and defaults to `false`. It is set to `true` here as the example DAGs will be used when verifying the installation.
+- the `spec.exposeConfig` key is optional and defaults to `false`. It is set to `true` only as an aid to verify the configuration and should never be used as such in anything other than test or demo clusters.
+- the previously created secret must be referenced in `spec.credentialsSecret`
+
+
+NOTE: Please note that the version you need to specify for `spec.version` is not only the version of Apache Airflow which you want to roll out, but has to be amended with a Stackable version as shown. This Stackable version is the version of the underlying container image which is used to execute the processes. For a list of available versions please check our
+https://repo.stackable.tech/#browse/browse:docker:v2%2Fstackable%airflow%2Ftags[image registry].
+It should generally be safe to simply use the latest image version that is available.
+
+This will create the actual Airflow cluster.
+
+=== Initialization of the Airflow database
```

Comment:
```suggestion
== Initialization of the Airflow database
```

I think this should be one level higher, right?

Reply:
Yes - thanks!